src/flb_metrics_exporter.c

```diff
@@ -173,6 +173,7 @@ static int collect_metrics(struct flb_me *me)
     collect_inputs(&mp_sbuf, &mp_pck, me->config);
     collect_filters(&mp_sbuf, &mp_pck, me->config);
     collect_outputs(&mp_sbuf, &mp_pck, me->config);
+    // FIXME: is this necessary or are new metrics only exposed via the v2 api?
```

Comment:
The new metrics are not exposed via the v1 endpoints. Is that generally how we treat new metrics or should I also expose them through v1 too?